### DISCUSSION TITLE
Add an achievement transfer tool for admins

### DISCUSSION THREAD

**Discussion by wescopeland** [RetroAchievements/RAWeb#1806] [Ideas]
**Requirements**
* Be able to transfer unlocks to and from Unofficial achievements.
* Be able to copy unlock timestamps from source to target achievements.

**Nice to haves**
* Be able to batch unlocks in a sequence if the source achievement total matches the target achievement total.
* Be able to multiple source copy data to a single target (probably take the earliest date in this case) and vice versa.

**Additional context**
https://discord.com/channels/476211979464343552/1026595325038833725/1145369591363665990

**Comment by Jamiras** [comment]
Mostly implement in #2009. The "Be able to batch unlocks in a sequence if the source achievement total matches the target achievement total." conflicts with the ability to support multiple source or destination achievements.
